models/tables/FRA_estimates_table.txt

```diff
@@ -0,0 +1,23 @@
+Regression output: France		
```

Comment:
Do we need to save both rtf and txt? If so, feel free to just resolve this comment